src/util/parseCsv.web.ts

```diff
@@ -0,0 +1,19 @@
+const papaParser = require('papaparse');
+
+export function parseCSVToJSON(fileObject: File | string) {
```

Comment:
Was providing `string` here necessary? By default jsc8 runs on node runtime locally, so I thought that it wont complain for the browser function

Reply:
Yes. To make the functions uniform i have updated here as well

Reply:
If it is not required please don't make types less concrete. It is done as a workaround for the node runtime. If not required here please remove.

Reply:
@ajaykrishnareddy-n @lohaniabhishek My bad. @ajaykrishnareddy-n Please remove it.